api/api_sources/sources/database/database-schema/schema-sqls/ObservationSchema.sql

```diff
@@ -11,6 +11,16 @@ ALTER TABLE observation ADD COLUMN area_length NUMERIC(7, 2) NULL DEFAULT 0.0;
 ALTER TABLE observation ADD COLUMN access_description VARCHAR(500) NULL;
 ALTER TABLE observation ADD COLUMN observer_first_name VARCHAR(100) NULL;
 ALTER TABLE observation ADD COLUMN observer_last_name VARCHAR(100) NULL;
+ALTER TABLE observation ADD COLUMN sample_identifier VARCHAR(50) NULL;
```

Comment:
I am wondering about free form text fields for identifiers... it may be fine for now, but we should probably look at the business requirements for storing this information. If an identifier is created and maintained somewhere else (i.e. where is sample data going to come from?) then we should validate the data from an existing list.